parachain/src/command.rs

```diff
@@ -48,7 +48,7 @@ fn load_spec(id: &str, para_id: ParaId) -> std::result::Result<Box<dyn sc_servic
 
 impl SubstrateCli for Cli {
     fn impl_name() -> String {
-        "BTC Parachain".into()
+        "InterBTC Parachain".into()
```

Comment:
```suggestion
        "interBTC Parachain".into()
```

Lowercase i for consistency